ReactiveFeedback/FeedbackLoopLense.swift

```diff
@@ -0,0 +1,80 @@
+import ReactiveSwift
+
+@propertyWrapper
+public class FeedbackLoopLense<Value>: MutablePropertyProtocol {
```

Comment:
```suggestion
public class FeedbackLoopLens<Value>: MutablePropertyProtocol {
```

Not going to add suggestions to all the places where this needs changing (including the file name) 😄